src/acme/ui/AcmeJPanelBase.java

```diff
@@ -0,0 +1,38 @@
+package acme.ui;
+
+import acme.pd.Company;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
I got a strange error on importing this class:
![image](https://user-images.githubusercontent.com/18337623/41323579-e25b3ed4-6e74-11e8-8d6d-c90df2ed2572.png)

Since this isn't part of the standard Java 8 library, let's follow [this suggestion](https://stackoverflow.com/questions/16374189/notimplementedexception-is-internal-proprietary-api) and use `java.lang.UnsupportedOperationException`.

Reply:
Ya I just assumed that it had it, intellij apparently found some weird jar, good catch